src/components/search-panel/SearchPanelFilters.tsx

```diff
@@ -20,9 +20,7 @@ import {
 	FilterRowTogglerConfig,
 	FilterRowSwitcherConfig,
 } from '../../models/filter/FilterInputs';
-import { SSEFilterInfo } from '../../api/sse';
-import { SearchPanelType } from './SearchPanel';
-import { FiltersHistoryType } from '../../stores/FiltersHistoryStore';
+import '../../styles/filter.scss';
```

Comment:
this import is kind of implicit. why does this file import styles if it doesnt have any components?
and i believe removed this file in 6.1